src/pages/Editor/Editor.module.css

```
@@ -1,19 +1,90 @@
   overflow: hidden;
-}
-.svgContainer svg {
+  transition:
+    transform 180ms ease,
+    box-shadow 180ms ease;
+  min-height: 360px;
   width: 100%;
   height: 100%;
+  max-height: 90vh;
+  cursor: grab;
+  position: relative;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

position is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)